csbuild/_testing/pylint_test.py

```diff
@@ -70,8 +70,12 @@ def _parseAndRejigger(module, data):
 				if match:
 					out.append('  File "{}", line {}, in pylint\n    {}'.format(module, match.group(1), match.group(2)))
 				else:
-					out.append(line)
-			return "\n".join(out) + "\n"
+					# Recent versions of pylint annoyingly print these lines about the config file to stderr and while
+					# there is an option internally to silence these messages, there is no switch available externally
+					# for configuring it.  This means we need to check for those messages manually and discard them.
+					if not line.startswith("Using config file ") and line != "No config file found, using default configuration":
+						out.append(line)
```

Comment:
As mentioned, the option exists internally within pylint itself, but last I looked, they did not expose anything to let user code toggle it.  I can check again in the latest version of pylint, but even if they have addressed it, it should go only once we're ready to commit to that version, otherwise it will break the build.

Reply:
I'm not that worried about it. I just wanted to make sure you knew how we were doing it in case it was easy to fix with a parameter to run_pylint or something.